README.md

```diff
@@ -1,6 +1,8 @@
 Description
 -----------
 
+[![Join the chat at https://gitter.im/backdrop/backdrop](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/backdrop/backdrop?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Thanks for the update @gitter-badger ! Unfortunately we don't want this code at the very top of our readme. Could you please add it towards the bottom, perhaps with a section heading of it's own?